src_c/surface.c

```diff
@@ -1884,6 +1887,66 @@ surf_blit(pgSurfaceObject *self, PyObject *args, PyObject *keywds)
     return pgRect_New(&dest_rect);
 }
 
+static PyObject *
+surf_rblit(pgSurfaceObject *self, PyObject *const *args, Py_ssize_t nargs)
+{
+    SDL_Surface *src, *dest = pgSurface_AsSurface(self);
+    SDL_Rect *src_rect, dest_rect, temp;
+    PyObject *blit_pos, *src_surf;
+    int flags_numeric = 0;
+
+    if (nargs != 2 || nargs != 3) {
```

Comment:
```suggestion
    if (nargs != 2 && nargs != 3) {
```

The current check doesn't work.